src/appleseed.studio/mainwindow/rendering/qttilecallback.cpp

```diff
@@ -64,8 +65,8 @@ namespace
           : m_render_widget(render_widget)
         {
             connect(
-                this, SIGNAL(signal_update()),
-                m_render_widget, SLOT(update()),
+                this, &QtTileCallback::signal_update,
+                m_render_widget, QOverload<void>::of(&QWidget::update),
```

Comment:
Given that `QOverload` causes problems in `doubleslider.cpp`, I'm sure this one will do so as well (the compiler didn't get here yet, so we can't know for sure).